book/extending.qmd

```diff
@@ -30,6 +30,67 @@ The chapter [Create a new learner](#extending-learners) illustrates the steps ne
 
 {{< include _complex.qmd >}}
 
+
+## Creating a ParamSet {#sec-extending-paramset}
```

Comment:
Fundamentally this is not extending as you're not creating a new class but talking about constructing an object